manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -82,6 +82,7 @@ spec:
             runAsGroup: 0
             runAsUser: 0
             runAsNonRoot: false
+            privileged: {{ .Values.cni.privileged | quote }}
```

Comment:
is quote right? shouldn't it be `true` not `"true" `?

Reply:
You are right. I got this mixed up with env values.